app/scripts/filters/findProject.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .filter('findProject', function($filter) {
+    return function(items, searchString) {
+      // return early if no search string
+      if(!searchString) {
+        return items;
+      }
+
+      return _.filter(items, function(item) {
+        var searchStringLower = searchString.toLowerCase();
```

Comment:
Lets move the evaluation of the search string up above the filter as it only needs to be evaluated once.  
```javascript
var searchStringLower = _.lowerCase(searchString);
return _.filter(items, function(item) {
 // else it does the same work over and over...
```